awx/main/expect/isolated_manager.py

```diff
@@ -246,7 +250,7 @@ def build_isolated_job_data(self):
         os.makedirs(self.path_to('artifacts', 'job_events'), mode=stat.S_IXUSR + stat.S_IWUSR + stat.S_IRUSR)
 
     def _missing_artifacts(self, path_list, output):
-        missing_artifacts = filter(lambda path: not os.path.exists(path), path_list)
+        missing_artifacts = list(filter(lambda path: not os.path.exists(path), path_list))
```

Comment:
`filter` already return an iterator. Maybe we can drop the list cast here.

Reply:
There are len/truthiness checks later in code, so it was easier to just make it a list.